code/modules/mob/living/carbon/xenomorph/strains/castes/boiler/trapper.dm

```diff
@@ -50,6 +50,16 @@
 	var/temp_movespeed_usable = FALSE
 	var/temp_movespeed_messaged = FALSE
 
+/datum/behavior_delegate/boiler_trapper/append_to_stat()
+	. = list()
+	var/datum/action/xeno_action/activable/boiler_trap/trap_ability = get_action(bound_xeno, /datum/action/xeno_action/activable/boiler_trap)
+	. += "Insight Amount: [trap_ability.empowering_charge_counter]/[trap_ability.empower_charge_max]"
+	if(trap_ability.empowered)
+		. += "Your next [trap_ability.name] will be empowered."
+	var/datum/action/xeno_action/activable/acid_mine/mine_ability = get_action(bound_xeno, /datum/action/xeno_action/activable/acid_mine)
+	if(mine_ability.empowered)
+		. += "Your next [mine_ability.name] will be empowered."
```

Comment:
Do we really need to append to stat since theres already a text message and an ui indicator now, im not sure if cluttering up the pannel further is good

Reply:
looks fine otherwise

Reply:
i think knowing how much insight you currently have is somewhat essential, likewise any other "stack" meters are present in stat for other strains
Berserker rage, acid, etc

Reply:
Got it, i'll then remove that entry